Tasks/Common/azure-arm-rest/azure-arm-app-service-kudu.ts

```diff
@@ -37,7 +37,7 @@ export class KuduServiceManagementClient {
                         tl.warning(tl.loc('ASE_SSLIssueRecommendation'));
                 }
 
-                if(retryCount > 0 && exceptionString.indexOf('Request timeout') != -1 && reqOptions.retryRequestTimedout) {
+                if(retryCount > 0 && exceptionString.indexOf('Request timeout') != -1 && reqOptions && reqOptions.retryRequestTimedout) {
```

Comment:
Do we need to increase task version?

Reply:
the task version is already increased in previous PR. so skipping here.